src/altinn-app-frontend/src/components/message/ErrorReport.tsx

```diff
@@ -1,128 +1,145 @@
 import * as React from 'react';
 
-import { useAppSelector } from 'src/common/hooks';
-import { getUnmappedErrors } from 'src/utils/validation';
-import type { IValidations } from 'src/types';
+import { Panel, PanelVariant } from '@altinn/altinn-design-system';
+import { Grid, makeStyles } from '@material-ui/core';
+
+import { useAppDispatch, useAppSelector } from 'src/common/hooks';
+import { FullWidthWrapper } from 'src/features/form/components/FullWidthWrapper';
+import { renderLayoutComponent } from 'src/features/form/containers/Form';
+import { FormLayoutActions } from 'src/features/form/layout/formLayoutSlice';
+import { getMappedErrors, getUnmappedErrors } from 'src/utils/validation';
+import type { ILayout } from 'src/features/form/layout';
+import type { FlatError } from 'src/utils/validation';
 
 import { getLanguageFromKey } from 'altinn-shared/utils';
 
-const ErrorReport = () => {
-  const validations = useAppSelector(
-    (state) => state.formValidations.validations,
+export interface IErrorReportProps {
+  components: ILayout;
+}
+
+const ArrowForwardIcon =
+  '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" style="position: relative; top: 2px">' +
+  '<path d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"></path>' +
+  '</svg>';
```

Comment:
Would suggest using a template literal here

```suggestion
const ArrowForwardIcon = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" style="position: relative; top: 2px">
  <path d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"></path>
</svg>`;
```

That should give a bit nicer syntax highlighting (at least in some editors) and you can easily write with the correct indentation. Though its not really important 😅